completion/models.py

```diff
@@ -168,6 +168,38 @@ class BlockCompletion(TimeStampedModel, models.Model):
 
     objects = BlockCompletionManager()
 
+    @classmethod
+    def get_course_completions(cls, user, course_key):
+        """
+        query all completions for course/user pair
+
+        Return value:
+            dict[BlockKey] = float
+        """
+        course_block_completions = cls.objects.filter(
+            user=user,
+            course_key=course_key,
+        )
+        # will not return if <= 0.0
```

Comment:
What does this comment mean?  Is it true?

Reply:
@iloveagent57  It's true in practical terms (as of now on platform) -- but it's not true _by necessity_. I'll ditch the comment.